starter_code/src/movies.js

```diff
@@ -1,9 +1,25 @@
-/* eslint no-restricted-globals: 'off' */
+// /* eslint no-restricted-globals: 'off' *
 // Turn duration of the movies from hours to minutes 
-
-
+function turnHoursToMinutes(array){
+    var firstNum;
+    var secondNum;
+    return array.map(function(movies){
+        var firstInt = movies.duration.split(' ');
+        var firstNum = parseInt(firstInt[0].slice(0, 1) * 60);
+        var secondNum = parseInt(firstInt[1].slice(0, 2));
+    return Object.assign({}, movies, {duration: firstNum + secondNum})
+    })  
+}
+console.log(turnHoursToMinutes(movies));
 // Get the average of all rates with 2 decimals 
 
```

Comment:
First iteration is working properly. Please keep working on this exercise and don't hesitate to ask us for any help you need.